packages/validator-superstruct/src/index.ts

```diff
@@ -24,7 +24,7 @@ export function validateStruct<Data extends Obj>(
   }
   return function validate(values: Data): Errors<Data> | undefined {
     try {
-      struct.assert(values);
+      struct.create(values);
```

Comment:
This would not update the values on our `data` store to the coerced result. I am guessing that's a desired behaviour?